File: src/main/java/ru/igrey/dev/domain/Cas.java
package ru.igrey.dev.domain;

import com.google.gson.annotations.SerializedName;
import lombok.Data;

import java.util.List;

@Data
public class Cas {
    @SerializedName(value = "casId")
    private Long id;
    private String phone;
    private String firstName;
    private String lastName;
    private String middleName;
    private List<String> authorities;

    private String fio() {
        return lastName + " " + firstName + " " + middleName;
    }

    @Override
    public String toString() {
        return "casId: " + id + ", phone: " + phone + ", " + fio();// + " authorities=" + authorities;
    }
}

File: src/main/kotlin/ru/igrey/dev/domain/Cas.kt
package ru.igrey.dev.domain

import com.google.gson.annotations.SerializedName

data class Cas(
    @SerializedName(value = "casId")
    val id: Long,
    val phone: String?,
    val firstName: String?,
    val lastName: String?,
    val middleName: String?
) {
    private fun fio(): String {
        return this.lastName + " " + this.firstName + " " + this.middleName
    }

    override fun toString(): String {
        return "casId: " + this.id + ", phone: " + this.phone + ", " + fio()// + " authorities=" + authorities;
    }
}

data class CasUsers(
    private val casUsers: List<Cas> = emptyList()
) {

    override fun toString(): String {
        if (casUsers.isEmpty()) return "EMPTY"
        val sb = StringBuilder()
        for (cas in casUsers) {
            sb.append(cas.toString()).append("\n")
        }
        return sb.toString()
    }
}